integration-tests/README.md

```diff
@@ -31,18 +31,50 @@ docker info
 Running Integration tests
 =========================
 
-## Running tests using mvn
+## Starting docker tests
 
-To run all the tests using mvn run the following command -
+To run all the tests using docker and mvn run the following command -
 '''''
   mvn verify -P integration-tests
 '''''
 
-To run only a single test using mvn run following command -
+To run only a single test using mvn run the following command -
 '''''
   mvn verify -P integration-tests -Dit.test=<test_name>
 '''''
 
+## Configure and run integration tests using existing cluster
```

Comment:
Also, ITIndexerTest uses baseDir "/resources/indexer", which is present because docker setup mounts it.  I was thinking of making the baseDir a test configuration parameter.